### PR TITLE
1.x: Release 1.1.2 CHANGES.md update

### PR DESCRIPTION
**Pull request by akarnokd**
Preparing for release 1.1.2.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+  - [Pull 3759](https://github.com/ReactiveX/RxJava/pull/3759): Add `Observable.concatDelayError` and `Observable.concatMapDelayError` operators
+  - [Pull 3763](https://github.com/ReactiveX/RxJava/pull/3763): Add `Observable.combineLatestDelayError` operator
+  - [Pull 3752](https://github.com/ReactiveX/RxJava/pull/3752): Add `Single.using` operator
+  - [Pull 3722](https://github.com/ReactiveX/RxJava/pull/3722): Add `Observable.flatMapIterable` overload with `maxConcurrent` parameter
+  - [Pull 3741](https://github.com/ReactiveX/RxJava/pull/3741): Add `Single.doOnSubscribe` operator
+  - [Pull 3738](https://github.com/ReactiveX/RxJava/pull/3738): Add `Observable.create(SyncOnSubscribe)` and `Observable.create(AsyncOnSubscribe)` factory methods
+  - [Pull 3718](https://github.com/ReactiveX/RxJava/pull/3718): Add `Observable.concatMapIterable` operator
+  - [Pull 3712](https://github.com/ReactiveX/RxJava/pull/3712): Add `Single.takeUntil(Completable)` operator
+  - [Pull 3696](https://github.com/ReactiveX/RxJava/pull/3696): Added Single execution hooks via `RxJavaSingleExecutionHook` class
```
Comment (artem-zinnatullin) [inline CHANGES.md]: This PR introduced little binary incompatible [change](https://github.com/ReactiveX/RxJava/pull/3696#discussion_r53476625), maybe worth mention here?

Comment (akarnokd) [inline CHANGES.md]: Updated.

**Comment by artem-zinnatullin** [comment]
:+1:

**Comment by zsxwing** [comment]
:+1:

**Comment by stevegury** [comment]
:+1:

**Pull request merged**